context_request.go

```diff
@@ -34,12 +42,16 @@ type ContextRequest struct {
 }
 
 func NewContextRequest(ctx *Context, log log.Log, validation contractsvalidate.Validation) contractshttp.ContextRequest {
+	request := contextRequestPool.Get().(*ContextRequest)
 	httpBody, err := getHttpBody(ctx)
 	if err != nil {
-		LogFacade.Error(fmt.Sprintf("%+v", err))
+		log.Error(fmt.Sprintf("%+v", errors.Unwrap(err)))
```

Comment:
`Unwrap` can't return correct error in some cases.

```suggestion
		log.Error(fmt.Sprintf("%+v", err))
```